install/gcp/installer/forseti_server_installer.py

```diff
@@ -101,16 +106,19 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             instance_name = 'forseti-{}-vm-{}'.format(
                 self.config.installation_type,
                 self.config.identifier)
-            self.wait_until_vm_initialized(instance_name)
 
             # Create firewall rules.
             self.create_firewall_rules()
+            self.wait_until_vm_initialized(instance_name)
+
+            # Delete firewall rules.
+            self.delete_firewall_rules()
```

Comment:
These default rules are not server specific, and should be moved outside of the server installer.

Reply:
It gives VM initialization error if logic to delete default rules is in forseti_installer.py because the rules will be deleted before the server and the client rules are created.

Reply:
Hmmm, okay.  I suppose we can can say that the correct firewalls are created only if the server is updated.

Otherwise, the relevant place to do this more centrally would be here.
https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/install/gcp/gcp_installer.py#L122-L137

Reply:
Based on the flow of control, I have put the logic in server as all the service account rules has to be created before default rules are deleted.

Reply:
Okay